File: core/src/com/unciv/models/stats/CivStats.java
package com.unciv.models.stats;

public class CivStats {
    public float gold = 0;
    public float science = 0;
    public float culture = 0;
    public float happiness = 0;

    public void add(CivStats other) {
        gold += other.gold;
        science += other.science;
        happiness += other.happiness;
        culture += other.culture;
    }
}

File: core/src/com/unciv/models/stats/CivStats.kt
package com.unciv.models.stats

open class CivStats {
    @JvmField  var gold = 0f
    @JvmField  var science = 0f
    @JvmField  var culture = 0f
    @JvmField  var happiness = 0f

    fun add(other: CivStats) {
        gold += other.gold
        science += other.science
        happiness += other.happiness
        culture += other.culture
    }
}
